tests/jballerina-unit-test/src/test/resources/test-src/bala/test_projects/test_project_records/closed_record_type_reference.bal

```diff
@@ -39,3 +39,8 @@ public type ClosedVehicleWithNever record {|
     int j;
     never p?;
 |};
+
+public type BClosedStudent record {|
+    string name = "";
+    int age = 0;
```

Comment:
Can we use different values rather than the defaults, just to make sure default values aren't used unintentionally from the types?